gc/base/GCExtensionsBase.cpp

```diff
@@ -244,7 +249,7 @@ void
 MM_GCExtensionsBase::tearDown(MM_EnvironmentBase* env)
 {
 #if defined(OMR_GC_MODRON_SCAVENGER) || defined(OMR_GC_VLHGC)
-	scavengerHotFieldStats.tearDown(env);
+	scavengerHotFieldStats->kill(this);
```

Comment:
missed check for (NULL != _hotFieldStats)

Reply:
👍